core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -707,6 +714,14 @@ class ZkUtils(val zkClient: ZkClient,
     }
   }
 
+  def getTopicPartitionCount(topic: String, defaultNumPartitions: Int = 50): Int = {
```

Comment:
Thinking about this a bit more. I felt the default number is weird anyways, will remove this parameter and still let each class as a little bit duplicate logic of using the default config values.